src/main/java/Duke.java

```diff
@@ -24,13 +27,18 @@ public void run() {
         sayGreeting();
         String command;
         Scanner sc = new Scanner(System.in);
-        while (true) {
+        do {
             command = sc.nextLine();
-            if (command.equals("bye")) {
+            switch (command) {
+            case "list":
+                printTasks();
+                break;
+            case "bye":
                 break;
+            default:
```

Comment:
Would it be better to place the addTask() function under one of the cases ("add") instead of under the default clause? Because I think that following coding standards, "default:" should be used to catch any input that's "out of the norm", instead of being used to process a valid input!

Reply:
Oh thanks for pointing this out! Did not realize that rule in the coding standards.